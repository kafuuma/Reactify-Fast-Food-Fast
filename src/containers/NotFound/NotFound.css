.not-found-display {
  font-size: 72;
  font-weight: "bold";
}
