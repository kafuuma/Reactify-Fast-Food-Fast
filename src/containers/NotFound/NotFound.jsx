import React from 'react';
import { MDBContainer } from 'mdbreact';
import notFound from '../../assets/notFound.png';
import './NotFound.css';

const NotFound = () => {
  return (
    <MDBContainer className="w-75 mt-5 pt-5">
      <div style={{ textAlign: 'center' }}>
        <h1>Page Not Found</h1>
        <img src={notFound} alt="NotFoud" />
      </div>
    </MDBContainer>
  );
};
export default NotFound;
