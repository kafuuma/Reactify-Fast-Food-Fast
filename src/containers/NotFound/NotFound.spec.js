import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('<NotFound />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });

  it('matches snapshort', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
