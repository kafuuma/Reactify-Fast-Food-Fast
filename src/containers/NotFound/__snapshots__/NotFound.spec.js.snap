// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> matches snapshort 1`] = `ShallowWrapper {}`;
